manage_breast_screening/participants/migrations/0035_implantedmedicaldevicehistoryitem.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 5.2.7 on 2025-11-10 11:19
+
+import django.db.models.deletion
+import uuid
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('participants', '0034_breastcancerhistoryitem'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ImplantedMedicalDeviceHistoryItem',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('device', models.CharField(choices=[('CARDIAC_DEVICE', 'Cardiac device (such as a pacemaker or ICD)'), ('HICKMAN_LINE', 'Hickman line'), ('OTHER_MEDICAL_DEVICE', 'Other medical device')])),
+                ('device_other_medical_device', models.CharField(blank=True, default='')),
```

Comment:
Should this be `other_medical_device_details`, for consistency with other details fields that we've implemented?